src/helpers/assetHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { Asset } from '@thorchain/asgardex-util'
+import { sequenceS } from 'fp-ts/lib/Apply'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+
+export const unsafeAssetToString = (asset: Asset): string => {
+  const { chain, symbol } = asset
+  return `${chain}.${symbol}`
+}
+
+const sequenseSO = sequenceS(O.option)
+
+export const assetToString = (asset: Asset): O.Option<string> => {
```

Comment:
dont you think that here is a semantic missmatch ?
Option<string> does not look like string 😅

Reply:
Agree, it needs to be improved. As mentioned before, we have to think about current implementation of `Asset` in `@thorchain/asgardex-util` again. Also I will extract this helper into this package.

Reply:
Will be fixed w/ https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/issues/9